client/src/components/mapping/InputStatic/index.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  Button,
+  Card,
+  ControlGroup,
+  Elevation,
+  InputGroup,
+  Position
+} from '@blueprintjs/core';
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useMutation, useQuery } from '@apollo/react-hooks';
+import { loader } from 'graphql.macro';
+import { ResourceDefinition } from '@arkhn/fhir.ts';
+
+import { onError as onApolloError } from 'services/apollo';
+import { IInput, IReduxStore } from 'types';
+
+import StringSelect from 'components/selects/stringSelect';
+import IdentifierSystemInput from './IdentifierSystemInput';
+
+// GRAPHQL
+const qSourcesAndResources = loader(
+  'src/graphql/queries/sourcesAndResources.graphql'
+);
+const mUpdateStaticInput = loader(
+  'src/graphql/mutations/updateStaticInput.graphql'
+);
+const mDeleteInput = loader('src/graphql/mutations/deleteInput.graphql');
+
+interface Props {
+  input: IInput;
+}
+
+const InputStatic = ({ input }: Props): React.ReactElement => {
+  const toaster = useSelector((state: IReduxStore) => state.toaster);
+  const onError = onApolloError(toaster);
+
+  const { attribute } = useSelector((state: IReduxStore) => state.selectedNode);
+  const { availableResources } = useSelector(
+    (state: IReduxStore) => state.fhir
+  );
+
+  const [staticValue, setStaticValue] = useState(input.staticValue);
+
+  const { data: dataSources } = useQuery(qSourcesAndResources, {
+    fetchPolicy: 'no-cache'
+  });
+  const [updateStaticInput] = useMutation(mUpdateStaticInput, { onError });
+  const [deleteInput, { loading: loadDelInput }] = useMutation(mDeleteInput, {
+    onError
+  });
+
+  const sources = dataSources ? dataSources.sources : [];
+
+  useEffect(() => {
+    setStaticValue(input.staticValue);
+  }, [input]);
+
+  const onUpdate = async (value: string): Promise<void> => {
```

Comment:
`async` keyword is useless here

Unless you forgot to use `await`before `updateStaticInput`